web/themes/new_weather_theme/templates/partials/daily-forecast-list-item.html.twig

```diff
@@ -20,24 +20,27 @@ only be a nighttime period.
         <span class="display-block">{{ periods[0].monthAndDay }}</span>
       </time>
 
-      {% for period in periods %}
-        {% include '@new_weather_theme/partials/daily-high-low.html.twig' with { 'period': period } %}
-      {% endfor %}
+      {% include '@new_weather_theme/partials/daily-high-low.html.twig' with { 'periods': periods } %}
     </div>
     <div class="grid-row grid-gap">
-      {% for period in periods %}
-        {% if period.isDaytime %}
-          {% set label = "Day" | t %}
-          {% set lineColor = "yellow" %}
-          {% set marginUnit = 3 %}
-        {% else %}
-          {% set label = "Night" | t %}
-          {% set lineColor = "indigo" %}
-          {% set marginUnit = 1 %}
-        {% endif %}
-        <div class="daily-forecast-summary-area grid-col-12 tablet:grid-col-6 margin-bottom-{{ marginUnit }} tablet:margin-bottom-1">
+        {% for period in periods %}
+            {% set marginUnit = 1 %}
+            {% set tabletCol = 6 %}
+            {%  if period.isOvernight %}
+                {% set label = "Overnight" | t %}
+                {% set lineColor = "indigo" %}
```

Comment:
For Overnight I was thinking we'd use a slightly different shade to differentiate the line from the Night period. But we would need to add it to the CSS. I am thinking we could change  the value `lineColor` here to `midnight-indigo` and add this css rule to the set of `line-before`s in `_uswds-theme-custom-styles.scss`
 

```
.line-before--midnight-indigo {
  &::before {
    border-top: 2px solid color("secondary-darker");
  }
}
```